apisix/http/router/radixtree_uri.lua

```diff
@@ -38,6 +38,11 @@ local function create_radixtree_router(routes)
 
     for _, route in ipairs(routes) do
         if type(route) == "table" then
+            local status = core.table.try_read_attr(route, "value", "status")
+            if status and status ~= 1 then
```

Comment:
we can use `if status == 0 then` here, it seems clearer

Reply:
solve